### ISSUE TITLE
Show more informative NAT status

### ISSUE THREAD

**Issue by interro** [mysteriumnetwork/dvpn-web#48]
NAT status: show more informative message if possible. For example _success_ or reason why it can’t be opened.

**Comment by ignasbernotas** [comment]
I don't think we can show "Waiting for incoming connections..." because the pending state depends on which NAT mechanism is being used. This would only work for NAT hole punching, but I don't think we can differentiate. @interro

**Comment by interro** [comment]
@zolia
